### DISCUSSION TITLE
Aqara SP-EUC01 reporting interval/reporting change

### DISCUSSION THREAD

**Discussion by KHV8** [Koenkk/zigbee2mqtt#16184] [Devices]
Hi,

I have a Aqara SP-EUC01 which works as expected. How, I have a problem. I report on energy/power way to often, on all changes it seems. As it is controlling a plug where there are several electronic devices, then the up/down changes on the devices are constant, changing in the range from 40w to 52w, going a little up and down. This leads to my most reporting devices.

I would like to change the reporting interval, and have looked into the reporting interval which is 5 sec as default. I have tried to change it to 60sec, however there is no changes. It reports every 5 sec. I have also tried to change the "minimum reporting change" to 20, as it then should not report on smaller changes in power whitout getting any changes at all. As I read somewhere this is in milliwatt, I have also tried with 20000, no change.

Any good ideas of how to solve this. In HA this leads to the 4 entities taking most space in my DB, as I use the value in a utility sensor, a history sensor and a simple sensor doing some math.

**Comment by KHV8** [comment]
Have made some additional test and reading. I found this describtion on in the documentation of how to do this, however not able to make it work. Just trying with increasing the minimum reporting interval, however not sure im doing this the correct way.

`zigbee2mqtt/bridge/request/device/configure_reporting
Allows to send a Zigbee configure reporting command to a device. Refer to the Configure Reporting Command in the [ZigBee Cluster Libraryopen in new window](https://github.com/Koenkk/zigbee-herdsman/blob/master/docs/07-5123-08-Zigbee-Cluster-Library.pdf) for more information. Example payload is {"id":"my_bulb","cluster":"genLevelCtrl","attribute":"currentLevel","minimum_report_interval":5,"maximum_report_interval":10,"reportable_change":10}. In this case the response would`


I get this error message from Z2M when I try.
Error 2023-01-17 21:27:57Request 'zigbee2mqtt/bridge/request/device/configure_reporting' failed with error: 'ConfigureReporting 0x04cf8cdf3c79d402/1 genLevelCtrl([{"attribute":"currentLevel","minimumReportInterval":60,"maximumReportInterval":3600,"reportableChange":0}], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 51419 - 1 - 67 - 8 - 7 after 10000ms)'

**Comment by KHV8** [comment]
I still do not have any luck. The devices are flooding my network (not creating instability) and I would like to limit the reporting. 
Anybody, any idea of where I can search and what to try? Or, if you know this is not possible, please let me hear

**Comment by frlequ** [comment]
Have the same issue but in reverse. I want to change the SP-EUC01 update power reading interval to 1 minute, but it seems that's not possible. 

I'm only able to apply new settings to currentSummDelivered... but it's still pulling updates with default value of 5min. 

Any solution?

**Reply by KHV8** [reply]
No, I have not got any input or been able to change this. 

Strange your device only report ebery 5 min, if just I could get my devices to do this.

**Comment by frlequ** [comment]
Weird. Just noticed that 2/3 of my SP-EUC01 devices reports once every 5min, but one of them reports every minute.
Also: On the third one, there is no Voltage and Current displayed in z2m

They all have the same Firmware Version 41 build date 12-06-2021.

**Reply by KHV8** [reply]
Mine is 12-6-2021 except for the 2 new. They are on 09-22-2020, not sure why they do not show up on the OTA list.

**Comment by djderex** [comment]
I have the same issue in 2026 with this same plug

**Reply by Trurl101** [reply]
> I have the same issue in 2026 with this same plug

Thanks for the Info. I am looking for new smart plugs and considered the Aqara, but not anymore.

I bought Third Reality Smart Plugs that flooded the network too, the newest inoffical firmware fixed the bug, but its not even officially released: https://github.com/Koenkk/zigbee2mqtt/issues/31332

Perhaps Aqara is using standard Tuya plugs too.
